3rd-party-licenses.md

```diff
@@ -0,0 +1,147 @@
+# License overview of included 3rd party libraries
+
+Shopsys Framework is licensed under the terms of the [Shopsys Community License](./LICENSE).
+
+Shopsys Framework also uses some third-party components and images
+which are licensed under their own respective licenses.
+
+## Components used by Shopsys Framework
+### Symfony Framework and Symfony Components
+License: MIT  
+https://symfony.com/
+
+### Elasticsearch
+License: Apache License 2.0  
+https://github.com/elastic/elasticsearch-php/blob/master/LICENSE
+
+### Grunt: The JavaScript Task Runner
+License: MIT  
+https://github.com/gruntjs/grunt
+
+### Phing
+License: LGPL-3.0-only  
+https://github.com/phingofficial/phing/blob/master/LICENSE
+
+### Nette Foundation tools 
+License: BSD-2-Clause or GPL-2.0 or GPL-3.0  
+https://nette.org/
```

Comment:
There is no licence info - I thought you always link the page with the actual licence, not just a homepage.

Reply:
Fixed